packages/cojson-storage-indexeddb/src/syncManager.ts

```diff
@@ -87,12 +87,14 @@ export class SyncManager {
 
     // reverse it to send the top level id the last in the order
     const collectedMessages = Object.values(outputMessages).reverse();
-    collectedMessages.forEach(({ knownMessage, contentMessages }) => {
+    for (const { knownMessage, contentMessages } of collectedMessages) {
       this.sendStateMessage(knownMessage);
-
-      contentMessages?.length &&
-        contentMessages.forEach((msg) => this.sendStateMessage(msg));
-    });
+      if (contentMessages?.length) {
+        for (const msg of contentMessages) {
+          this.sendStateMessage(msg);
+        }
+      }
+    }
```

Comment:
This one as well - two nested blocks vs zero and necessity to declare a variable another time. Of course this is about coding style which is a personal thing. Some might not like `&&` but still - `for ... of` is something I would't expect in JS a lot